presto-main/src/main/java/com/facebook/presto/execution/scheduler/NodeScheduler.java

```diff
@@ -263,9 +263,11 @@ private Supplier<NodeMap> createNodeMapSupplier(ConnectorId connectorId)
                     .map(InternalNode::getNodeIdentifier)
                     .collect(toImmutableSet());
 
-            int weight = (int) ceil(1.0 * minVirtualNodeCount / activeNodes.size());
-            ConsistentHashingNodeProvider consistentHashingNodeProvider = ConsistentHashingNodeProvider.create(activeNodes, weight);
-
+            ConsistentHashingNodeProvider consistentHashingNodeProvider = null;
```

Comment:
In this case, let's make this `Optional` rather than using null.

Reply:
Done